CONTRIBUTING.md

```diff
@@ -1,19 +1,17 @@
-# OpenSDS
+# Soda
 
-[![Go Report Card](https://goreportcard.com/badge/github.com/opensds/opensds?branch=master)](https://goreportcard.com/report/github.com/opensds/opensds)
-[![Build Status](https://travis-ci.org/opensds/opensds.svg?branch=master)](https://travis-ci.org/opensds/opensds)
-[![Coverage Status](https://coveralls.io/repos/github/opensds/opensds/badge.svg?branch=master)](https://coveralls.io/github/opensds/opensds?branch=master)
+[![Coverage Status](https://coveralls.io/repos/github/soda/soda/badge.svg?branch=master)](https://coveralls.io/github/soda/soda?branch=master)
 
 <img src="https://www.opensds.io/wp-content/uploads/sites/18/2016/11/logo_opensds.png" width="100">
```

Comment:
This image can be changed to the appropriate logo image from the [SODA Foundation website](https://sodafoundation.io)